editor/components/url-input/button.js

```diff
@@ -76,4 +77,17 @@ class UrlInputButton extends Component {
 	}
 }
 
-export default UrlInputButton;
+export class UrlInputButton extends URLInputButton {
+	constructor() {
+		super( ...arguments );
+
+		deprecated( 'wp.editor.UrlInputButton', {
+			alternative: 'wp.editor.URLInputButton',
+			plugin: 'Gutenberg',
+			version: 'v3.4',
```

Comment:
This code will be released on 3.3 should the deprecation version be 3.5?

Reply:
Yes, it should be `3.5`.